src/DynamoCore/Graph/Nodes/NodeCategories.cs

```diff
@@ -420,8 +421,8 @@ internal static string MakeRelativePath(string basePath, string subjectPath)
             var documentUri = new Uri(basePath, UriKind.Absolute);
             var assemblyUri = new Uri(subjectPath, UriKind.Absolute);
 
-            var relativeUri = documentUri.MakeRelativeUri(assemblyUri);
-            var relativePath = relativeUri.OriginalString.Replace('/', '\\');
+            var relativeUri = documentUri.MakeRelativeUri(assemblyUri).OriginalString;
+            var relativePath = WebUtility.UrlDecode(relativeUri.Replace("+", "%2B")).Replace('/', '\\');
```

Comment:
@gregmarr thanks for checking mono build - just to clarify this was with our internal build version, not travis or something else with 4.0 or up?